libnlopt-dev system dependency undocumented in Quick Start and Build from Source guides

**Description:** No pre-built nlopt wheel exists for linux/aarch64 on PyPI, so pip falls back to building from source. That build requires libnlopt-dev to be installed on the system, but it is not listed as a prerequisite in either the [https://nvidia.github.io/IsaacTeleop/main/getting_started/build_from_source/index.htm](https://nvidia.github.io/IsaacTeleop/main/getting_started/quick_start.html)or [https://nvidia.github.io/IsaacTeleop/main/getting_started/build_from_source/index.htm](https://nvidia.github.io/IsaacTeleop/main/getting_started/build_from_source/index.htm) docs.

**To reproduce**
Steps to reproduce the behavior:
Follow the Quick Start guide: pip install 'isaacteleop[cloudxr,retargeters]~=1.0.0' --extra-index-url https://pypi.nvidia.com/

**Expected behavior**
Package installs successfully

**Observed behavior**
Build fails with could not build wheels for nlopt

**Screenshots**
<img width="1613" height="1325" alt="Image" src="https://github.com/user-attachments/assets/85f1c2a9-a6cb-4b38-ae7b-b3925c9d40f2" />
<img width="1618" height="1346" alt="Image" src="https://github.com/user-attachments/assets/e4f4b98a-7454-4fa8-94ca-807317cffca8" />
<img width="1621" height="1279" alt="Image" src="https://github.com/user-attachments/assets/2f7fb813-d3cf-486d-b066-88a4cd2bc0b2" />

**Platforms**
 - Isaac Teleop Version: N/A
 - OS: aarch64 (NVIDIA DGX Spark, Ubuntu)
 - Integrations: Isaac Teleop
 - XR Device: N/A

**Additional context**
I ran the steps on the build from source page, but I am getting multiple errors. It's not finding the c library, so the python wheel can't be built.

Can repro

```
$ uv pip install 'isaacteleop[cloudxr,retargeters]~=1.0.0' --extra-index-url https://pypi.nvidia.com
Resolved 62 packages in 1.54s
  × Failed to build `nlopt==2.6.2`
  ├─▶ The build backend returned an error
  ╰─▶ Call to `setuptools.build_meta:__legacy__.build_wheel` failed (exit status: 1)

      [stdout]
      running bdist_wheel
      running build
      running build_ext
      cmake version 3.28.3

      CMake suite maintained and supported by Kitware (kitware.com/cmake).
      -- The C compiler identification is GNU 13.3.0
      -- The CXX compiler identification is GNU 13.3.0
      -- Detecting C compiler ABI info
      -- Detecting C compiler ABI info - done
      -- Check for working C compiler: /usr/bin/cc - skipped
      -- Detecting C compile features
      -- Detecting C compile features - done
      -- Detecting CXX compiler ABI info
      -- Detecting CXX compiler ABI info - done
      -- Check for working CXX compiler: /usr/bin/c++ - skipped
      -- Detecting CXX compile features
      -- Detecting CXX compile features - done
      -- Found PythonInterp: /var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/bin/python (found version "3.12.3")
      -- Found Python includes: /usr/include/python3.12
      -- Found Python libs: /usr/lib/aarch64-linux-gnu
      -- NLopt version 2.6.2
      -- Looking for dlfcn.h
      -- Looking for dlfcn.h - found
      -- Looking for getopt.h
      -- Looking for getopt.h - found
      -- Looking for unistd.h
      -- Looking for unistd.h - found
      -- Looking for string.h
      -- Looking for string.h - found
      -- Looking for strings.h
      -- Looking for strings.h - found
      -- Looking for inttypes.h
      -- Looking for inttypes.h - found
      -- Looking for memory.h
      -- Looking for memory.h - found
      -- Looking for stdlib.h
      -- Looking for stdlib.h - found
      -- Looking for stdint.h
      -- Looking for stdint.h - found
      -- Looking for time.h
      -- Looking for time.h - found
      -- Looking for sys/types.h
      -- Looking for sys/types.h - found
      -- Looking for sys/stat.h
      -- Looking for sys/stat.h - found
      -- Looking for sys/time.h
      -- Looking for sys/time.h - found
      -- Looking for getpid
      -- Looking for getpid - found
      -- Looking for syscall
      -- Looking for syscall - found
      -- Looking for isinf
      -- Looking for isinf - found
      -- Looking for isnan
      -- Looking for isnan - found
      -- Looking for gettimeofday
      -- Looking for gettimeofday - found
      -- Looking for qsort_r
      -- Looking for qsort_r - found
      -- Looking for time
      -- Looking for time - found
      -- Looking for copysign
      -- Looking for copysign - found
      -- Looking for stddef.h
      -- Looking for stddef.h - found
      -- Check size of uint32_t
      -- Check size of uint32_t - done
      -- Check size of unsigned int
      -- Check size of unsigned int - done
      -- Check size of unsigned long
      -- Check size of unsigned long - done
      -- Looking for sqrt in m
      -- Looking for sqrt in m - found
      -- Looking for fpclassify
      -- Looking for fpclassify - TRUE
      -- Performing Test HAVE_THREAD_LOCAL_STORAGE
      -- Performing Test HAVE_THREAD_LOCAL_STORAGE - Success
      -- Performing Test HAVE_THREAD_LOCAL_STORAGE
      -- Performing Test HAVE_THREAD_LOCAL_STORAGE - Failed
      -- Looking for __cplusplus
      -- Looking for __cplusplus - found
      -- Performing Test SUPPORTS_STDCXX11
      -- Performing Test SUPPORTS_STDCXX11 - Success
      -- Performing Test HAS_FPIC
      -- Performing Test HAS_FPIC - Success
      -- Found PythonLibs: /usr/lib/aarch64-linux-gnu (Required is exact version "3.12.3")
      -- Could NOT find NumPy (missing: NUMPY_INCLUDE_DIRS)
      -- Could NOT find Guile (missing: GUILE_EXECUTABLE GUILE_ROOT_DIR GUILE_INCLUDE_DIRS GUILE_LIBRARIES)
      -- Found SWIG: /usr/bin/swig4.0 (found version "4.2.0")
      -- Performing Test UNDEFINED_SYMBOLS_ALLOWED - Success
      -- Could NOT find Octave (missing: OCTAVE_EXECUTABLE OCTAVE_ROOT_DIR OCTAVE_INCLUDE_DIRS OCTAVE_LIBRARIES)
      -- Could NOT find Matlab (missing: Matlab_INCLUDE_DIRS Matlab_MEX_LIBRARY Matlab_MEX_EXTENSION Matlab_ROOT_DIR Matlab_MX_LIBRARY MX_LIBRARY MAIN_PROGRAM) (found version
      "NOTFOUND")
      -- Configuring done (2.7s)
      -- Generating done (0.0s)
      -- Build files have been written to: /var/local/home/horde/.cache/uv/sdists-v9/pypi/nlopt/2.6.2/PUD889lLsirHM49zWGKq3/src/build/temp.linux-aarch64-cpython-312
      [  1%] Generating nlopt.hpp
      [  3%] Generating nlopt.f
      [  3%] Built target generate-fortran
      [  3%] Built target generate-cpp
      [  5%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/direct/direct_wrap.c.o
      [  7%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/direct/DIRect.c.o
      [  9%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/direct/DIRserial.c.o
      [ 11%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/direct/DIRsubrout.c.o
      [ 13%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/cdirect/cdirect.c.o
      [ 15%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/cdirect/hybrid.c.o
      [ 17%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/praxis/praxis.c.o
      [ 19%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/luksan/plis.c.o
      [ 21%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/luksan/plip.c.o
      [ 23%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/luksan/pnet.c.o
      [ 25%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/luksan/mssubs.c.o
      [ 27%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/luksan/pssubs.c.o
      [ 29%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/crs/crs.c.o
      [ 31%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/mlsl/mlsl.c.o
      [ 33%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/mma/mma.c.o
      [ 35%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/mma/ccsa_quadratic.c.o
      [ 37%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/cobyla/cobyla.c.o
      [ 39%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/newuoa/newuoa.c.o
      [ 41%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/neldermead/nldrmd.c.o
      [ 43%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/neldermead/sbplx.c.o
      [ 45%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/auglag/auglag.c.o
      [ 47%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/bobyqa/bobyqa.c.o
      [ 49%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/isres/isres.c.o
      [ 50%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/slsqp/slsqp.c.o
      [ 52%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/esch/esch.c.o
      [ 54%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/api/general.c.o
      [ 56%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/api/options.c.o
      [ 58%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/api/optimize.c.o
      [ 60%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/api/deprecated.c.o
      [ 62%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/api/f77api.c.o
      [ 64%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/util/mt19937ar.c.o
      [ 66%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/util/sobolseq.c.o
      [ 68%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/util/timer.c.o
      [ 70%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/util/stop.c.o
      [ 72%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/util/redblack.c.o
      [ 74%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/util/qsort_r.c.o
      [ 76%] Building C object extern/nlopt/CMakeFiles/nlopt.dir/src/util/rescale.c.o
      [ 78%] Building CXX object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/stogo/global.cc.o
      [ 80%] Building CXX object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/stogo/linalg.cc.o
      [ 82%] Building CXX object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/stogo/local.cc.o
      [ 84%] Building CXX object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/stogo/stogo.cc.o
      [ 86%] Building CXX object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/stogo/tools.cc.o
      [ 88%] Building CXX object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/ags/evolvent.cc.o
      [ 90%] Building CXX object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/ags/solver.cc.o
      [ 92%] Building CXX object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/ags/local_optimizer.cc.o
      [ 94%] Building CXX object extern/nlopt/CMakeFiles/nlopt.dir/src/algs/ags/ags.cc.o
      [ 96%] Linking CXX static library libnlopt.a
      [100%] Built target nlopt

      [stderr]
      CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
        Compatibility with CMake < 3.5 will be removed from a future version of
        CMake.

        Update the VERSION argument <min> value or use a ...<max> suffix to tell
        CMake that the project does not need compatibility with older versions.


      CMake Warning (dev) at CMakeLists.txt:12 (find_package):
        Policy CMP0148 is not set: The FindPythonInterp and FindPythonLibs modules
        are removed.  Run "cmake --help-policy CMP0148" for policy details.  Use
        the cmake_policy command to set the policy and suppress this warning.

      This warning is for project developers.  Use -Wno-dev to suppress it.

      CMake Deprecation Warning at extern/nlopt/CMakeLists.txt:15 (cmake_minimum_required):
        Compatibility with CMake < 3.5 will be removed from a future version of
        CMake.

        Update the VERSION argument <min> value or use a ...<max> suffix to tell
        CMake that the project does not need compatibility with older versions.


      CMake Warning (dev) at extern/nlopt/CMakeLists.txt:306 (find_package):
        Policy CMP0148 is not set: The FindPythonInterp and FindPythonLibs modules
        are removed.  Run "cmake --help-policy CMP0148" for policy details.  Use
        the cmake_policy command to set the policy and suppress this warning.

      This warning is for project developers.  Use -Wno-dev to suppress it.

      CMake Warning (dev) at extern/nlopt/CMakeLists.txt:307 (find_package):
        Policy CMP0148 is not set: The FindPythonInterp and FindPythonLibs modules
        are removed.  Run "cmake --help-policy CMP0148" for policy details.  Use
        the cmake_policy command to set the policy and suppress this warning.

      This warning is for project developers.  Use -Wno-dev to suppress it.

      CMake Warning (dev) at extern/nlopt/cmake/FindNumPy.cmake:45 (find_package):
        Policy CMP0148 is not set: The FindPythonInterp and FindPythonLibs modules
        are removed.  Run "cmake --help-policy CMP0148" for policy details.  Use
        the cmake_policy command to set the policy and suppress this warning.

      Call Stack (most recent call first):
        extern/nlopt/CMakeLists.txt:308 (find_package)
      This warning is for project developers.  Use -Wno-dev to suppress it.

      CMake Warning (dev) at
      /var/local/home/horde/.cache/uv/sdists-v9/pypi/nlopt/2.6.2/PUD889lLsirHM49zWGKq3/src/build/temp.linux-aarch64-cpython-312/extern/nlopt/CMakeTmp/UNDEFINED_SYMBOLS_ALLOWED/CMakeLists.txt:2
      (project):
        cmake_minimum_required() should be called prior to this top-level project()
        call.  Please see the cmake-commands(7) manual for usage documentation of
        both commands.
      This warning is for project developers.  Use -Wno-dev to suppress it.

      CMake Deprecation Warning at /var/local/home/horde/.cache/uv/sdists-v9/pypi/nlopt/2.6.2/PUD889lLsirHM49zWGKq3/src/extern/nlopt/cmake/generate-cpp.cmake:1
      (cmake_minimum_required):
        Compatibility with CMake < 3.5 will be removed from a future version of
        CMake.

        Update the VERSION argument <min> value or use a ...<max> suffix to tell
        CMake that the project does not need compatibility with older versions.


      CMake Deprecation Warning at /var/local/home/horde/.cache/uv/sdists-v9/pypi/nlopt/2.6.2/PUD889lLsirHM49zWGKq3/src/extern/nlopt/cmake/generate-fortran.cmake:1
      (cmake_minimum_required):
        Compatibility with CMake < 3.5 will be removed from a future version of
        CMake.

        Update the VERSION argument <min> value or use a ...<max> suffix to tell
        CMake that the project does not need compatibility with older versions.


      In file included from /usr/include/string.h:548,
                       from /var/local/home/horde/.cache/uv/sdists-v9/pypi/nlopt/2.6.2/PUD889lLsirHM49zWGKq3/src/extern/nlopt/src/algs/neldermead/nldrmd.c:25:
      In function ‘memset’,
          inlined from ‘nldrmd_minimize_’ at /var/local/home/horde/.cache/uv/sdists-v9/pypi/nlopt/2.6.2/PUD889lLsirHM49zWGKq3/src/extern/nlopt/src/algs/neldermead/nldrmd.c:205:4:
      /usr/include/aarch64-linux-gnu/bits/string_fortified.h:59:10: warning: ‘__builtin_memset’ specified bound between 18446744056529682432 and 18446744073709551608 exceeds
      maximum object size 9223372036854775807 [-Wstringop-overflow=]
         59 |   return __builtin___memset_chk (__dest, __ch, __len,
            |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
         60 |                                  __glibc_objsize0 (__dest));
            |                                  ~~~~~~~~~~~~~~~~~~~~~~~~~~
      Traceback (most recent call last):
        File "<string>", line 11, in <module>
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/build_meta.py", line 438, in build_wheel
          return _build(['bdist_wheel'])
                 ^^^^^^^^^^^^^^^^^^^^^^^
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/build_meta.py", line 429, in _build
          return self._build_with_temp_dir(
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/build_meta.py", line 410, in _build_with_temp_dir
          self.run_setup()
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/build_meta.py", line 520, in run_setup
          super().run_setup(setup_script=setup_script)
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/build_meta.py", line 317, in run_setup
          exec(code, locals())
        File "<string>", line 85, in <module>
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/__init__.py", line 117, in setup
          return distutils.core.setup(**attrs)  # type: ignore[return-value]
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/_distutils/core.py", line 186, in setup
          return run_commands(dist)
                 ^^^^^^^^^^^^^^^^^^
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/_distutils/core.py", line 202, in run_commands
          dist.run_commands()
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/_distutils/dist.py", line 1000, in run_commands
          self.run_command(cmd)
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/dist.py", line 1107, in run_command
          super().run_command(command)
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/_distutils/dist.py", line 1019, in run_command
          cmd_obj.run()
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/command/bdist_wheel.py", line 370, in run
          self.run_command("build")
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/_distutils/cmd.py", line 341, in run_command
          self.distribution.run_command(command)
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/dist.py", line 1107, in run_command
          super().run_command(command)
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/_distutils/dist.py", line 1019, in run_command
          cmd_obj.run()
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/_distutils/command/build.py", line 135, in run
          self.run_command(cmd_name)
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/_distutils/cmd.py", line 341, in run_command
          self.distribution.run_command(command)
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/dist.py", line 1107, in run_command
          super().run_command(command)
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpikzWzm/lib/python3.12/site-packages/setuptools/_distutils/dist.py", line 1019, in run_command
          cmd_obj.run()
        File "<string>", line 28, in run
        File "<string>", line 70, in build_extension
      StopIteration

      hint: This usually indicates a problem with the package or the build environment.
  help: `nlopt` (v2.6.2) was included because `isaacteleop[retargeters]` (v1.0.193) depends on `nlopt`
(IsaacTeleop) horde@p4242-0323:~/yanziz/IsaacTeleop$ uv pip install 'isaacteleop[cloudxr,retargeters]~=1.0.0' --extra-index-url https://pypi.nvidia.com
Resolved 62 packages in 764ms
  × Failed to build `nlopt==2.6.2`
  ├─▶ The build backend returned an error
  ╰─▶ Call to `setuptools.build_meta:__legacy__.build_wheel` failed (exit status: 1)

      [stdout]
      running bdist_wheel
      running build
      running build_ext
      cmake version 3.28.3

      CMake suite maintained and supported by Kitware (kitware.com/cmake).
      -- Found PythonInterp: /var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/bin/python (found version "3.12.3")
      -- Found Python includes: /usr/include/python3.12
      -- Found Python libs: /usr/lib/aarch64-linux-gnu
      -- NLopt version 2.6.2
      -- Could NOT find NumPy (missing: NUMPY_INCLUDE_DIRS)
      -- Could NOT find Guile (missing: GUILE_EXECUTABLE GUILE_ROOT_DIR GUILE_INCLUDE_DIRS GUILE_LIBRARIES)
      -- Could NOT find Octave (missing: OCTAVE_EXECUTABLE OCTAVE_ROOT_DIR OCTAVE_INCLUDE_DIRS OCTAVE_LIBRARIES)
      -- Could NOT find Matlab (missing: Matlab_INCLUDE_DIRS Matlab_MEX_LIBRARY Matlab_MEX_EXTENSION Matlab_ROOT_DIR Matlab_MX_LIBRARY MX_LIBRARY MAIN_PROGRAM) (found version
      "NOTFOUND")
      -- Configuring done (0.2s)
      -- Generating done (0.0s)
      -- Build files have been written to: /var/local/home/horde/.cache/uv/sdists-v9/pypi/nlopt/2.6.2/PUD889lLsirHM49zWGKq3/src/build/temp.linux-aarch64-cpython-312
      [  1%] Built target generate-fortran
      [  3%] Built target generate-cpp
      [100%] Built target nlopt

      [stderr]
      CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
        Compatibility with CMake < 3.5 will be removed from a future version of
        CMake.

        Update the VERSION argument <min> value or use a ...<max> suffix to tell
        CMake that the project does not need compatibility with older versions.


      CMake Warning (dev) at CMakeLists.txt:12 (find_package):
        Policy CMP0148 is not set: The FindPythonInterp and FindPythonLibs modules
        are removed.  Run "cmake --help-policy CMP0148" for policy details.  Use
        the cmake_policy command to set the policy and suppress this warning.

      This warning is for project developers.  Use -Wno-dev to suppress it.

      CMake Deprecation Warning at extern/nlopt/CMakeLists.txt:15 (cmake_minimum_required):
        Compatibility with CMake < 3.5 will be removed from a future version of
        CMake.

        Update the VERSION argument <min> value or use a ...<max> suffix to tell
        CMake that the project does not need compatibility with older versions.


      CMake Warning (dev) at extern/nlopt/CMakeLists.txt:306 (find_package):
        Policy CMP0148 is not set: The FindPythonInterp and FindPythonLibs modules
        are removed.  Run "cmake --help-policy CMP0148" for policy details.  Use
        the cmake_policy command to set the policy and suppress this warning.

      This warning is for project developers.  Use -Wno-dev to suppress it.

      CMake Warning (dev) at extern/nlopt/CMakeLists.txt:307 (find_package):
        Policy CMP0148 is not set: The FindPythonInterp and FindPythonLibs modules
        are removed.  Run "cmake --help-policy CMP0148" for policy details.  Use
        the cmake_policy command to set the policy and suppress this warning.

      This warning is for project developers.  Use -Wno-dev to suppress it.

      CMake Warning (dev) at extern/nlopt/cmake/FindNumPy.cmake:45 (find_package):
        Policy CMP0148 is not set: The FindPythonInterp and FindPythonLibs modules
        are removed.  Run "cmake --help-policy CMP0148" for policy details.  Use
        the cmake_policy command to set the policy and suppress this warning.

      Call Stack (most recent call first):
        extern/nlopt/CMakeLists.txt:308 (find_package)
      This warning is for project developers.  Use -Wno-dev to suppress it.

      Traceback (most recent call last):
        File "<string>", line 11, in <module>
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/build_meta.py", line 438, in build_wheel
          return _build(['bdist_wheel'])
                 ^^^^^^^^^^^^^^^^^^^^^^^
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/build_meta.py", line 429, in _build
          return self._build_with_temp_dir(
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/build_meta.py", line 410, in _build_with_temp_dir
          self.run_setup()
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/build_meta.py", line 520, in run_setup
          super().run_setup(setup_script=setup_script)
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/build_meta.py", line 317, in run_setup
          exec(code, locals())
        File "<string>", line 85, in <module>
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/__init__.py", line 117, in setup
          return distutils.core.setup(**attrs)  # type: ignore[return-value]
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/_distutils/core.py", line 186, in setup
          return run_commands(dist)
                 ^^^^^^^^^^^^^^^^^^
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/_distutils/core.py", line 202, in run_commands
          dist.run_commands()
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/_distutils/dist.py", line 1000, in run_commands
          self.run_command(cmd)
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/dist.py", line 1107, in run_command
          super().run_command(command)
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/_distutils/dist.py", line 1019, in run_command
          cmd_obj.run()
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/command/bdist_wheel.py", line 370, in run
          self.run_command("build")
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/_distutils/cmd.py", line 341, in run_command
          self.distribution.run_command(command)
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/dist.py", line 1107, in run_command
          super().run_command(command)
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/_distutils/dist.py", line 1019, in run_command
          cmd_obj.run()
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/_distutils/command/build.py", line 135, in run
          self.run_command(cmd_name)
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/_distutils/cmd.py", line 341, in run_command
          self.distribution.run_command(command)
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/dist.py", line 1107, in run_command
          super().run_command(command)
        File "/var/local/home/horde/.cache/uv/builds-v0/.tmpKF5edg/lib/python3.12/site-packages/setuptools/_distutils/dist.py", line 1019, in run_command
          cmd_obj.run()
        File "<string>", line 28, in run
        File "<string>", line 70, in build_extension
      StopIteration

      hint: This usually indicates a problem with the package or the build environment.
  help: `nlopt` (v2.6.2) was included because `isaacteleop[retargeters]` (v1.0.193) depends on `nlopt`
```

--- a/docs/source/getting_started/build_from_source/index.rst
+++ b/docs/source/getting_started/build_from_source/index.rst
@@ -290,7 +290,8 @@ based on the Python version.  Note that ``pip`` and ``uv pip`` has slightly diff
       cd /tmp/nlopt-python
       git submodule update --init --recursive
 
-      uv venv --python=${PYTHON_VERSION} /tmp/nlopt-wheel-venv
+      # Use the same Python version as your isaacteleop install (3.10, 3.11, 3.12, or 3.13)
+      uv venv --python=3.11 /tmp/nlopt-wheel-venv
       VIRTUAL_ENV=/tmp/nlopt-wheel-venv uv pip install numpy setuptools wheel
       /tmp/nlopt-wheel-venv/bin/python setup.py bdist_wheel -d /tmp/nlopt-wheels/
 

--- a/docs/source/getting_started/build_from_source/index.rst
+++ b/docs/source/getting_started/build_from_source/index.rst
@@ -272,16 +272,55 @@ The wheels are built in the ``./install/wheels/`` directory. Install the package
 Using ``pip``, you need to pass the ``--no-index`` option to automatically find the right wheel
 based on the Python version.  Note that ``pip`` and ``uv pip`` has slightly different options.
 
+.. _aarch64-nlopt-wheel:
+
+.. note::
+   **ARM64 / aarch64 systems only** (e.g. NVIDIA DGX Spark): PyPI does not publish pre-built
+   ``nlopt`` wheels for ARM64, so pip cannot satisfy the ``retargeters`` extra automatically.
+   Build an ``nlopt`` wheel from source before running the install commands below
+   (see `issue #452 <https://github.com/NVIDIA/IsaacTeleop/issues/452>`_):
+
+   .. code-block:: bash
+
+      # Install build tools
+      sudo apt-get install -y build-essential cmake git pkg-config swig
+
+      # Clone nlopt-python and build a wheel
+      git clone --depth 1 --branch 2.10.0 https://github.com/DanielBok/nlopt-python.git /tmp/nlopt-python
+      cd /tmp/nlopt-python
+      git submodule update --init --recursive
+
+      uv venv --python=${PYTHON_VERSION} /tmp/nlopt-wheel-venv
+      VIRTUAL_ENV=/tmp/nlopt-wheel-venv uv pip install numpy setuptools wheel
+      /tmp/nlopt-wheel-venv/bin/python setup.py bdist_wheel -d /tmp/nlopt-wheels/
+
+   Then pass ``--find-links=/tmp/nlopt-wheels/`` when installing so the locally-built wheel is
+   used instead of attempting a PyPI download:
+
+   .. code-block:: bash
+
+      # pip
+      pip install "isaacteleop[retargeters,cloudxr,ui]" \
+          --find-links=./install/wheels/ \
+          --find-links=/tmp/nlopt-wheels/ \
+          --no-index --force-reinstall
+
+      # uv pip
+      uv pip install "isaacteleop[retargeters,cloudxr,ui]" \
+          --find-links=./install/wheels/ \
+          --find-links=/tmp/nlopt-wheels/ \
+          --reinstall
+
 .. code-block:: bash
 
    # Pass --no-index to use only wheels in ./install/wheels/;
    # Pass --force-reinstall to replace an existing install.
-   pip install isaacteleop[retargeters,cloudxr,ui] --find-links=./install/wheels/ --no-index --force-reinstall
+   pip install "isaacteleop[retargeters,cloudxr,ui]" --find-links=./install/wheels/ --no-index --force-reinstall
 
 .. code-block:: bash
 
    # Pass --reinstall to replace an existing install.
-   uv pip install isaacteleop[retargeters,cloudxr,ui] --find-links=./install/wheels/ --reinstall
+   uv pip install "isaacteleop[retargeters,cloudxr,ui]" --find-links=./install/wheels/ --reinstall
 
 .. toctree::
    :hidden:

--- a/docs/source/getting_started/quick_start.rst
+++ b/docs/source/getting_started/quick_start.rst
@@ -38,6 +38,19 @@ In a new terminal, install the package from PyPI (or from a local wheel if you b
 Instead of installing the package from PyPI, you can build from source and install the local wheel.
 See :doc:`build_from_source/index` for more details.
 
+.. note::
+   **ARM64 / aarch64 systems only** (e.g. NVIDIA DGX Spark): PyPI does not publish pre-built
+   ``nlopt`` wheels for ARM64, so the ``retargeters`` extra cannot be installed directly from PyPI
+   (see `issue #452 <https://github.com/NVIDIA/IsaacTeleop/issues/452>`_). Follow the
+   :ref:`aarch64 nlopt wheel build steps <aarch64-nlopt-wheel>` from the build-from-source guide
+   first, then install ``isaacteleop`` with an additional ``--find-links``:
+
+   .. code-block:: bash
+
+      pip install 'isaacteleop[cloudxr,retargeters]~=1.0.0' \
+          --extra-index-url https://pypi.nvidia.com \
+          --find-links=/tmp/nlopt-wheels/
+
 .. _run-cloudxr-server:
 
 3. Run CloudXR Server

